docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '3.0'
+
+services:
+  web:
+    image: rails_api_base
+    build: .
+    entrypoint: >
+      bash -c "
+      rm -f tmp/pids/server.pid
+      && bundle exec rails s -b 0.0.0.0 -p 3000"
```

Comment:
Won't this be affected in some way by Dockerfile:51?

Reply:
What happens in this case, is that the docker-compose's entrypoint overrides the one present in the Dockerfile. Because of this, if the developer wants to change the entrypoint later, there will not be any need to rebuild the image, just changing the entrypoint in the docker-compose will suffice.

Reply:
Make sense, thanks for the clear answer 👍